plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/impl/sql/StandardSQLDialectQueryGenerator.java

```diff
@@ -178,6 +181,39 @@ public void appendConditionString(
         }
     }
 
+    @Override
+    public String addFiltersToQuery(
+        @Nullable DBRProgressMonitor monitor,
+        @NotNull DBPDataSource dataSource,
+        @NotNull String sqlQuery,
+        @NotNull DBDDataFilter dataFilter
+    ) {
+        return dataSource.getSQLDialect().getQueryGenerator().addFiltersToQuery(monitor, dataSource, sqlQuery, dataFilter);
+    }
+
+
+    @Override
+    public String wrapQuery(
+        @NotNull DBPDataSource dataSource,
+        @NotNull String sqlQuery,
+        @NotNull DBDDataFilter dataFilter
+    ) {
+        // Append filter conditions to query
```

Comment:
It should be obvious from the method itself, not from some comment between its code